<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>skinwrapClass</class>
 <widget class="QDialog" name="skinwrapClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>skinwrap</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="pbDark">
     <property name="text">
      <string>dark</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbLight">
     <property name="text">
      <string>light</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelClose">
     <property name="text">
      <string>close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="skinwrap.qrc"/>
 </resources>
 <connections/>
</ui>
